shared-bindings/audiomp3/MP3Decoder.c

```diff
@@ -256,6 +276,7 @@ MP_PROPERTY_GETTER(audiomp3_mp3file_samples_decoded_obj,
 
 STATIC const mp_rom_map_elem_t audiomp3_mp3file_locals_dict_table[] = {
     // Methods
+    { MP_ROM_QSTR(MP_QSTR_open_file), MP_ROM_PTR(&audiomp3_mp3file_open_file_obj) },
```

Comment:
I thought maybe this would be `mp3file.open`, `open_file` feels a bit like a redundant name.